server/src/main/webapp/WEB-INF/rails/webpack/views/pages/pipelines/material_auto_update_toggle.tsx

```diff
@@ -32,19 +32,22 @@ interface Attrs {
 
 export class MaterialAutoUpdateToggle extends MithrilViewComponent<Attrs> {
   view(vnode: m.Vnode<Attrs>) {
-    const { noun, toggle, errors } = withDefaults(vnode.attrs);
-    const autoUpdate = this.radioState(toggle);
+    const {noun, toggle, errors} = withDefaults(vnode.attrs);
+    const autoUpdate             = this.radioState(toggle);
 
-    return <RadioField<AutoUpdateRadioOptions>
-      label={`${_.capitalize(noun)} polling behavior:`}
-      property={autoUpdate}
-      readonly={!!vnode.attrs.disabled}
-      errorText={errors?.errorsForDisplay("autoUpdate")}
-      dataTestId="material-auto-update"
-      possibleValues={[
-        {label: `Regularly fetch updates to this ${noun}`, value: "auto"},
-        {label: `Fetch updates to this ${noun} only on webhook or manual trigger`, value: "manual"},
-      ]}/>;
+    return [
+      <RadioField<AutoUpdateRadioOptions>
+        label={`${_.capitalize(noun)} polling behavior:`}
+        property={autoUpdate}
+        readonly={!!vnode.attrs.disabled}
+        errorText={errors?.errorsForDisplay("autoUpdate")}
+        dataTestId="material-auto-update"
+        possibleValues={[
+          {label: `Regularly fetch updates to this ${noun}`, value: "auto"},
+          {label: `Fetch updates to this ${noun} only on webhook or manual trigger`, value: "manual"},
+        ]}/>,
+      <ErrorText errorText={errors?.errorsForDisplay("autoUpdate")} errorId={`material-auto-update-error`}/>
```

Comment:
Hmm, looking at `RadioField`, it doesn't seem to render `errorText` on its own. Should we just add `<ErrorText/>` to `RadioField` like all the other input components? Seems like it may have been an oversight in its initial impl?

Reply:
Unless there is a good case for not showing errors normally, but I'd find that to be odd.

Reply:
Hmm sounds right to me. I will make the change.